mock-core-configs/mock-core-configs.spec

```diff
@@ -22,7 +22,7 @@ BuildArch:  noarch
 Provides: mock-configs
 
 # distribution-gpg-keys contains GPG keys used by mock configs
-Requires:   distribution-gpg-keys >= 1.112
+Requires:   distribution-gpg-keys >= 1.113
```

Comment:
Thank you for the bump here :100: 

Is the release scheduled?

Reply:
Hi there. Rocky Linux 10 was released June 11.